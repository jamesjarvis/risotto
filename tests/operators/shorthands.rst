a := 1
a += 3
println(a)

b := 1
b -= 3
println(b)

c := 1
c *= 3
println(c)

d := 1
d /= 3
println(d)

e := 1
e &= 3
println(e)

f := 1
f |= 3
println(f)

g := 1
g ^= 3
println(g)

h := 1
h <<= 3
println(h)

i := 1
i >>= 3
println(i)
