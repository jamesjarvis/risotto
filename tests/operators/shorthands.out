4
-2
3
0
1
3
2
8
0
